torch_ipex/csrc/cpu/dbl/Common.cpp

```diff
@@ -17,9 +17,14 @@ namespace dbl {
 namespace comm {
 
 dil::tensor dil_tensor_from_dense(const at::Tensor& tensor) {
-  AT_ASSERTM(
+  TORCH_CHECK(
     tensor.layout() == at::Layout::Strided,
     "dil_tensor_view_from_dense expects dense tensor input");
+  TORCH_CHECK(tensor.scalar_type() == at::ScalarType::Float ||
+             tensor.scalar_type() == at::ScalarType::BFloat16,
+             "dil_tensor_view_from_dense expects bfloat16 or float tensor input");
```

Comment:
error message was not correct

Reply:
What it should be? "dil_tensor_from_dense expects ......."?

Reply:
Yes. And the input scalar type should only be `at::ScalarType::Float`